nextflow.config

```diff
@@ -2,7 +2,7 @@
 vim: syntax=groovy
 -*- mode: groovy;-*-
  * -------------------------------------------------
- *  NGI-ExoSeq Nextflow config file
+ *  QBIC-ExoSeq Nextflow config file
```

Comment:
Confusing..?

Reply:
Changed in 4d1755a5e953293edf91e848c08a4b21a182c7c2